frontend/src/components/gotEnzymes/Landing.vue

```diff
@@ -76,18 +76,15 @@
             <TableOfContents :links="tocLinks" />
             <div id="intro" class="column content has-text-justified">
               <p>
-                GotEnzymes provides open access to over XYZ million predicted kinetic data points
-                for XYZ unique sequences and XYZ compounds from XYZ species with the cutting-edge
-                artificial intelligence tools. The values for kcat [reference] and KM [reference] in
-                this database are predicted from available advanced deep learning tools. Our vision
-                for this database is to store every possible predicted parameter for any available
-                enzyme to facilitate better science.
+                GotEnzymes provides open access to over 25,794,195 million predicted k<sub>cat</sub>
```

Comment:
A bit curious about this since we have >33 million rows in the `enzymes` table. If I count unique values (`select count(*) from enzymes group by kcat_values;`), we have ~1.8 million. It would be interesting to get some background about where this listed number comes from.

Reply:
I counted the unique proteinID&compund pairs. They may contain identical values, which is due to that protein sequences from different organisms are identifical.

Reply:
Ah I see. Thank you for taking time to explain!